File: core/src/main/java/com/egoal/darkestpixeldungeon/items/CrownOfDwarf.java
package com.egoal.darkestpixeldungeon.items;

import com.egoal.darkestpixeldungeon.sprites.ItemSprite;
import com.egoal.darkestpixeldungeon.sprites.ItemSpriteSheet;

/**
 * Created by 93942 on 9/24/2018.
 */

public class CrownOfDwarf extends Item {
  {
    image = ItemSpriteSheet.DPD_DWARF_CROWN;
  }

  public CrownOfDwarf() {
    super();
    identify();
  }

  @Override
  public boolean isUpgradable() {
    return false;
  }
  
  @Override
  public int price(){ return 750*quantity(); }

}

File: core/src/main/java/com/egoal/darkestpixeldungeon/items/CrownOfDwarf.kt
package com.egoal.darkestpixeldungeon.items

import com.egoal.darkestpixeldungeon.sprites.ItemSprite
import com.egoal.darkestpixeldungeon.sprites.ItemSpriteSheet

/**
 * Created by 93942 on 9/24/2018.
 */

class CrownOfDwarf : Item() {
    init {
        image = ItemSpriteSheet.DPD_DWARF_CROWN
        identify()
    }

    override fun isUpgradable() = false

    override fun price() = 750 * quantity()
}
